docs/DeveloperGuide.md

```diff
@@ -249,14 +249,17 @@ _{Explain here how the data archiving feature will be implemented}_
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
+| Priority | As a …​                                 | I want to …​                | So that I can…​                                                     |
 | -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| `* * *`  | new user                                   | see a list of available commands | refer to instructions when I forget how to use the App               |
+| `* * *`  | user                                       | create a budget book           | add new budgets book that i need                                       |
```

Comment:
I'm not sure if we want to use the term budget book compared to budgets. I used budgets in the User Guide, whichever case it should be consistent in both guides. @wenhaogoh please advise

Reply:
Yup, let's use "budget" instead of "budget books" as discussed. Good catch 👍

Reply:
okay will do